AGENTS.md

```
@@ -0,0 +1,295 @@
+- `parking_lot` — synchronization primitives
+- `moka` — block cache and vLog reader cache
+- `bytes` — zero-copy byte buffers
+- `crc32fast` — checksums
+- `farmhash` — bloom filter hashing
+- `nom` — CLI parser combinators
+- `ouroboros` — self-referencing structs
+- `criterion` — benchmarking
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)